gapis/api/vulkan/command_buffer_rebuilder.go

```diff
@@ -879,7 +879,7 @@ func rebuildVkCmdSetBlendConstants(
 	s *api.GlobalState,
 	d VkCmdSetBlendConstantsArgsʳ) (func(), api.Cmd, error) {
 
-	var constants F32ː4ᵃ
+	constants := NewF32ː4ᵃ(s.Arena)
```

Comment:
The constructor takes array values, so you can fold the next 4 lines into this one

Reply:
Changed to use the constructor for array values.